### ISSUE TITLE
Question about equilibration protocol 

### ISSUE THREAD

**Issue by jasmin-guven** [OpenBioSim/biosimspace_tutorials#29]
Hi! 

I was just refreshing my memory of the BSS syntax for minimisation and equilibration, and noticed [here](https://github.com/OpenBioSim/biosimspace_tutorials/blob/17bd74546004790cf075b5cffef9b0331bf2fddf/04_fep/02_RBFE/scripts/ligprep.py#L225) that the tutorials first have a restrained NVT step, followed by a non-restrained NVT step, which then is followed by a restrained NPT step. 

I wanted to ask if there is a reason why the position restraints are relaxed in between two position restraint steps? This seems different to similar tutorials from [Amber](https://ambermd.org/tutorials/basic/tutorial13/index.php#Minimize_the_added_water_and_ions_), for example. I've also had some collaborators also point this out to me in the past, so I thought I'd ask here. 

In my specific case of working with metals, the removal of restraints in between NVT and NPT allows the system to relax too much too early and can lead to problems with coordination. 

Thanks a lot in advance!

**Comment by jmichel80** [comment]
Hi @jasmin-guven I believe this protocol is based on @annamherz ‘s study . @annamherz can you comment ?

**Comment by jmichel80** [comment]
Just to add that if your use case warrants it it’s simple to just update locally the equilibration protocol. The RBFE tutorial illustrates how a BSS RBFE pipeline may be deployed but there’s no guarantee that the protocol is optimal for any input dataset.

**Comment by jasmin-guven** [comment]
> Just to add that if your use case warrants it it’s simple to just update locally the equilibration protocol. The RBFE tutorial illustrates how a BSS RBFE pipeline may be deployed but there’s no guarantee that the protocol is optimal for any input dataset.

Yes I see what you mean, thanks Julien! I was just curious to see if there was a specific reason behind this protocol 😊

**Comment by annamherz** [comment]
Hello 😊 I think this is the protocol I used at the start (I adapted it slightly more after with the restraint weights) but maybe as I was using the benchmarking systems I never experienced any issues with it, so I did not spend a lot of time optimising it after my systems were equilibrated. I agree that now I would probably run it without the release of the restraints at the end of the NVT step as I think that's more robust.

**Comment by lohedges** [comment]
Thanks for the clarification. We can leave this issue open as a reminder to rework the protocol for the next iteration of the tutorials.

**Comment by jasmin-guven** [comment]
Thanks everyone!
